### PR TITLE
Document new web-vitals attribution features

### PR DESCRIPTION
**Pull request by philipwalton**
This PR updates the [Debug Web Vitals in the field]() post to reference the new [attribution build](https://github.com/GoogleChrome/web-vitals#attribution-build) in `web-vitals` v3 and adds debug guidance for INP.

However, since the post now includes INP, to avoid any confusion (consistent with other recent changes we've made) this PR also updates the title and URL of the post from "Debug Web Vitals..." to "Debug performance...".

Note: to make it easier to review the diff I've kept the original file location (if I rename the file and make changes, GitHub does not render the diff properly, so it's impossible to review). Once these changes have been approved I'll update the file location and merge.

### CODE REVIEW

File: src/site/content/en/blog/debug-web-vitals-in-the-field/index.md [hunk 1]
```diff
@@ -190,20 +190,27 @@ This means you cannot make assumptions about which element or set of elements
 will be the most common LCP candidate element for a particular page. You have to
 measure it based on real-user behavior.
 
+{% Aside %}
+For more details on these differences, see: [Why lab and field data can be
+different (and what to do about it)](/lab-and-field-data-differences/)
+{% endAside %}
+
```
Comment (tunetheweb) [inline src/site/content/en/blog/debug-web-vitals-in-the-field/index.md:197]: Is it worth including a similar reference to https://web.dev/crux-and-rum-differences/ either here or elsewhere?

File: src/site/content/en/blog/debug-web-vitals-in-the-field/index.md [hunk 2]
```diff
@@ -277,157 +285,175 @@ function wasFIDBeforeDCL(fidEntry) {
...
+latency), knowing which elements your users are interacting with may be useful
+in determining how best to improve FID.
 
 For example, if the vast majority of your user's first interactions are with a
-particular element, it consider inlining the JavaScript code needed for that
+particular element, consider inlining the JavaScript code needed for that
 element in the HTML, and lazy loading the rest.
 
 To get the element associated with the first input event, you can reference the
 `first-input` entry's `target` property:
```
Comment (tunetheweb) [inline src/site/content/en/blog/debug-web-vitals-in-the-field/index.md:302]:
```suggestion
To get the element associated with the first input event, you can reference the
`first-input` entry's `target` and `name` properties:
```

**Review by tunetheweb — APPROVED** [review]
LGTM.

A couple of small suggestions.

Note this article is referenced (three times) in https://github.com/GoogleChrome/web-vitals/blob/main/README.md. I presume you'll add a redirect and also update that?

File: src/site/content/en/blog/debug-web-vitals-in-the-field/index.md (hunk 2 above)
Comment (philipwalton) [reply src/site/content/en/blog/debug-web-vitals-in-the-field/index.md:302]: Thanks, updated myself to also reference the name on 301.

**Comment by philipwalton** [comment]
> Note this article is referenced (three times) in https://github.com/GoogleChrome/web-vitals/blob/main/README.md. I presume you'll add a redirect and also update that?

Yeah, added.

**Pull request merged**